tap_stripe/__init__.py

```diff
@@ -698,6 +698,17 @@ def sync_event_updates(stream_name):
                 event_resource_metadata = metadata.to_map(
                     Context.get_catalog_entry(stream_name)['metadata']
                 )
+
+                # Filter out line items with null ids
+                if events_obj.get('data', {}).get('object', {}).get('lines', {}).get('data'):
```

Comment:
It might be better to check that it is indeed an invoice event here too. That should be on the event object somewhere.

It should work fine, unless another object type has `lines`, then it'll bleed out of the intended target space (e.g., if those lines aren't supposed to have IDs and aren't a separate stream).